### DISCUSSION TITLE
invalid character '<' looking for beginning of value

### DISCUSSION THREAD

**Discussion by alwaysbegood** [projectdiscovery/asnmap#261] [Q&A]
![image](https://github.com/projectdiscovery/asnmap/assets/20222407/5b466d24-f9a2-4d49-b638-6693ac56bc9f)

even use from manual installation or by released files, issue still the same

**Comment by k1ngpr4wn** [comment]
I had a similar error running `echo hackerone.com | asnmap`: "[FTL] invalid character 'e' looking for beginning of value"

After installing a few older versions, v1.0.5, v1.0.4, v1.0.3 (but getting a different panic error), then re-installing v1.0.6 again, it seems to have fixed something, and I don't see the first error anymore.

**Comment by Mzack9999** [comment]
See https://github.com/projectdiscovery/asnmap/discussions/260
